bundles/admin/admin-layereditor/components/Select.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from 'antd/lib/select';
```

Comment:
And do the same here. Rename Select -> AntSelect and export SelectComponent as Select.